contracts/ERC721M.sol

```diff
@@ -226,118 +207,35 @@ contract ERC721M is IERC721M, ERC721AQueryable, Ownable, ReentrancyGuard {
         return _maxMintableSupply;
     }
 
-    /**
-     * @dev Sets maximum mintable supply.
-     *
-     * New supply cannot be larger than the old.
-     */
-    function setMaxMintableSupply(uint256 maxMintableSupply)
```

Comment:
wait, I thought we need this:
<img width="1123" alt="image" src="https://github.com/magicoss/erc721m/assets/95395274/05d050df-c885-4ef5-a5d5-3a65a8083020">

Reply:
We have the https://github.com/magicoss/erc721m/blob/main/contracts/ERC721MIncreasableSupply.sol, cant we stick with that?

Reply:
ok, nvm, increasable doesnt allow cutting RIP, let me add it back.